front/admin/db.ts

```diff
@@ -36,7 +37,6 @@ import {
 } from "@app/lib/models";
 import {
   AgentTablesQueryAction,
```

Comment:
these imports should be from app/lib/models as well no?

Reply:
Yeah can fix. auto-import did its thing